packages/lambdas/src/fitbit.ts

```diff
@@ -0,0 +1,72 @@
+import { getSecret } from "@aws-lambda-powertools/parameters/secrets";
+import * as Sentry from "@sentry/serverless";
+import axios from "axios";
+import { createHmac } from "crypto";
+import { Request } from "express";
+import status from "http-status";
+import { capture } from "./shared/capture";
+import { getEnvOrFail } from "./shared/env";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const apiServerURL = getEnvOrFail("API_URL");
+const fitbitClientSecretName = getEnvOrFail("FITBIT_CLIENT_SECRET");
+
+const api = axios.create();
+
+const buildResponse = (status: number, body?: unknown) => ({
+  statusCode: status,
+  body,
+});
+
+const defaultResponse = () => buildResponse(status.NO_CONTENT);
+
+export const handler = Sentry.AWSLambda.wrapHandler(async (req: Request) => {
+  const secret: string = (await getSecret(fitbitClientSecretName)) as string;
+  if (!secret) {
+    throw new Error(`Config error - FITBIT_CLIENT_SECRET doesn't exist`);
+  }
+
+  if (!req.body) {
+    console.log("Request has no body - will not be forwarded to the API");
+    return defaultResponse();
+  }
```

Comment:
nit: could make this the first thing that happens, so if there's no body, we don't bother reading the secret